help/sources/tutorials/ui/create/cloud-storage/merkury-enterprise-identity-resolution.md

```diff
@@ -0,0 +1,448 @@
+
+  ---------------------------------------------------------------------------------
+  **Credential**   **Description**
+  ---------------- ----------------------------------------------------------------
+  AccessKey        The access key ID for your bucket.
+
+  SecretKey        The secret key ID for your bucket.
+
+  bucketName       The bucket contains your data and its corresponding descriptive
+                   metadata. Your S3 bucket name must be between three and 63
+                   characters long and must begin and end with either a letter or a
+                   number. The bucket name can only have lowercase letters,
+                   numbers, or hyphens (-), and cannot be formatted as an IP
+                   address.
+  ---------------------------------------------------------------------------------
+
+## Connect your Merkury Account
+
+In the Platform UI, select **Sources** from the left navigation bar to
+access the Sources workspace. The Catalog screen displays a variety of
+sources for which you can create an account with.
+
+You can select the Enrichment category from the catalog on the left-hand
+side of your screen. Alternatively, you can find the specific source you
+wish to work with using the search option.
+
+Under the **Enrichment** category, select **Merkury** and then
+select **Add data**.
+
+![](./merkury-enterprise-identity-resolution-assets/c5503baaad6e0be4a10cc675f6c6ddc5793d4dcb.png){width="6.349206036745406in"
+height="2.7777777777777777in"}
+
+The **Connect to Merkury** page appears. On this page, you can either
+use new credentials or existing credentials.
+
+New account
+
+If you are using new credentials, select **new account**. On the input
+form that appears, provide a name, an optional description, and your
+Merkury credentials. When finished, select **Connect to source** and
+then allow some time for the new connection to establish.
+
+![](./merkury-enterprise-identity-resolution-assets/cca42dc1b31c64075713b3814dd2d3b5ecc59da1.png){width="5.0in"
+height="4.40625in"}
+
+### Existing account
+
+To connect an existing account, select Merkury account you want to
+connect with, then select **Next** to proceed.
+
+![](./merkury-enterprise-identity-resolution-assets/616d2005eddb6ea8828572a523296d5c1172ed2e.png){width="6.943346456692914in"
+height="1.9962117235345582in"}
+
+**Configure a dataflow to ingest batch data from a cloud storage source
+in the UI**
+
+This tutorial provides steps on how to configure a dataflow to bring
+batch data from your source connector to Adobe Experience Platform.
+
+## Getting started
+
+**NOTE**
+
+In order to create a dataflow to bring batch data from a cloud storage,
+you must already have access to an authenticated cloud storage source.
+If you do not have access, go to the [sources
+overview](https://experienceleague.adobe.com/docs/experience-platform/sources/home.html?lang=en#cloud-storage) for
+a list of sources that you can create an account with.
```

Comment:
1. Since this is a new category (Enrichment), will these notes about cloud storage still apply?
2. If so, can we fix the formatting on the note? It should be:

>[!NOTE]
>
>Text here...